nesspoll/src/index.tsx

```diff
@@ -4,15 +4,16 @@ import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import registerServiceWorker from './registerServiceWorker';
 
 import Home from './wievs/Home';
+import Login from './wievs/Login';
```

Comment:
wievs? I think it should be called "views"

Reply:
will rename